<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckDlg</class>
 <widget class="QDialog" name="CheckDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 54, 61);
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.9, fx:0.5, fy:0.5, stop:0.0113636 rgba(85, 85, 85, 255), stop:0.403409 rgba(49, 49, 49, 255));</string>
  </property>
  <widget class="QPushButton" name="pushButton_Deposit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;微软雅黑&quot;;
color: rgb(80, 80, 80);
background-color: rgb(255, 255, 255);

border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>存款</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Withdrawl">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;微软雅黑&quot;;
color: rgb(80, 80, 80);
background-color: rgb(255, 255, 255);

border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>取款</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Transfer">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;微软雅黑&quot;;
color: rgb(80, 80, 80);
background-color: rgb(255, 255, 255);

border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>转账</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(7, 118, 209);
font: 75 16pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>   PZHU——ATM                                      客服热线 | 8888888</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;微软雅黑&quot;;
color: rgb(240, 240, 240);
background-color: rgb(2, 109, 189);

border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>退卡</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Exit_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;微软雅黑&quot;;
color: rgb(240, 240, 240);
background-color: rgb(2, 109, 189);

border:2px groove gray;
border-radius:10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>391</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 245, 245);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(7, 118, 209);
font: 75 16pt &quot;微软雅黑&quot;;
</string>
   </property>
   <property name="text">
    <string>您的账户余额信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 75 18pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;余 额：&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;可用余额：&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;ATM当日可体现余额：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>203</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 75 18pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);

border-width:0;
border-style:outset</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>246</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 75 18pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);


border-style:outset</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>288</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 75 18pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);

border-width:0;border-style:outset</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>pushButton_Deposit</zorder>
  <zorder>pushButton_Withdrawl</zorder>
  <zorder>pushButton_Transfer</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_Exit</zorder>
  <zorder>pushButton_Exit_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
 </widget>
 <tabstops>
  <tabstop>pushButton_Withdrawl</tabstop>
  <tabstop>pushButton_Transfer</tabstop>
  <tabstop>pushButton_Deposit</tabstop>
  <tabstop>pushButton_Exit_2</tabstop>
  <tabstop>pushButton_Exit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
